components/RepaymentInfo/RepaymentInfo.tsx

```diff
@@ -14,6 +17,37 @@ export function RepaymentInfo({ loan }: RepaymentInfoProps) {
     longFormattedInterestRate,
     longFormattedEstimatedPaybackAtMaturity,
   } = useLoanDetails(loan);
+
+  const maturityDate = useMemo(
+    () =>
+      loan.endDateTimestamp
+        ? dayjs.unix(loan.endDateTimestamp).format('MMMM D, YYYY')
+        : '',
+    [loan],
+  );
+
+  const createCalEvent = useCallback(() => {
+    const config: CalendarOptions = {
+      title: `Backed: Loan #${loan.id} ${loan.collateralName} due`,
+      description: `Loan repayment of ${longFormattedEstimatedPaybackAtMaturity} is due: https://www.withbacked.xyz/loans/${loan.id}`,
+      start: new Date(loan.endDateTimestamp! * 1000),
+    };
+
+    if (
+      /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
+        navigator.userAgent,
+      )
+    ) {
+      const icalendar = new ICalendar(config);
+      icalendar.download();
```

Comment:
on mobile, download ics event which will open default calendar app. we could also just default to google calendar link on mobile too though if we want? feel some people use apple calendar

Reply:
I think the assumption that everyone on desktop uses a google calendar is a bit suspect

Reply:
agree, probably had a bias towards myself. do you think we should just download an ics file regardless? maybe a modal that lets user choose between google calendar and ics file?

Reply:
could do a modal. maybe even just 2 different icons (if we can find a nice looking one for gcal)?

Reply:
i like two different icons